src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -14,28 +14,33 @@
  */
 public class DeleteCommandParser implements Parser<DeleteCommand> {
 
+    private boolean deleteParserCheck(String args, String commandType) throws ParseException {
+        if (args.startsWith(commandType)) {
```

Comment:
You can consider replacing this with a guard clause. However since its not very nested its not a pressing issue.